docs/components/InputFile/Web.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import {
+  FileUpload,
+  InputFile,
+  updateFiles,
+} from "@jobber/components/InputFile";
+import { FormatFile } from "@jobber/components/FormatFile";
+import { Heading } from "@jobber/components/Heading";
+
+export default {
+  title: "Components/Forms and Inputs/InputFile/Web",
+  component: InputFile,
+  parameters: {
+    viewMode: "story",
+    // CAN THIS BE SET FOR A PARTICULAR STORY?
```

Comment:
I'd love to be able to hide the controls for the `VariationsAndSizes` example. Anyone know how?